config/routes.rb

```diff
@@ -29,7 +29,7 @@
         via: %i[get post],
         format: :xml
 
-  post '/api/verify/images' => 'idv/image_uploads#create'
+  post '/api/verify/images' => 'idv/image_uploads#create', as: :verify_images_api
```

Comment:
For consistency, can we name this `api_verify_images`? It matches our other routes:

```
$ bundle exec rake routes | grep api
                              api_health GET             /api/health(.:format)                                                                    health/health#index
                     api_health_database GET             /api/health/database(.:format)                                                           health/database#index
                         api_health_jobs GET             /api/health/jobs(.:format)                                                               health/jobs#index
                api_openid_connect_certs GET             /api/openid_connect/certs(.:format)                                                      openid_connect/certs#index
                api_openid_connect_token POST            /api/openid_connect/token(.:format)                                                      openid_connect/token#create
...
````

Reply:
> For consistency, can we name this `api_verify_images`? It matches our other routes:

Yep, didn't see those, but it makes sense to be consistent 👍

Reply:
> > For consistency, can we name this `api_verify_images`? It matches our other routes:
> 
> Yep, didn't see those, but it makes sense to be consistent 👍

I'm guessing this is some Rails magic to implicitly create those names based on the path? In which case, it seems I could do away with `as:` altogether and still be able to reference it via `api_verify_images_url`.

Reply:
Updated in rebased a350e2765.

Reply:
✨ rails URL magic ✨ 

![shia labeuouf magic](https://media2.giphy.com/media/12NUbkX6p4xOO4/200.gif)